### DISCUSSION TITLE
Arrow Design change

### DISCUSSION THREAD

**Discussion by JoelHaimerl** [plantuml/plantuml#637] [Q&A] [answered]
Hallo, i want to change the Design of the Arrows in the TimelineDiagramm but i can`t find the correct line.
Can u pls help me with the Problem

PS: Sry for my bad Englsich

Joel

**Comment by The-Lum** [comment, marked as answer]
Hello @JoelHaimerl,

With the last beta-version _(V1.2021.10beta4+)_, you can now use style, as:
```puml
<style>
timingDiagram {
 constraintArrow {
  LineStyle 2-1
  LineThickness 3
  LineColor Blue
 }
}
</style>
```

>[![](http://www.plantuml.com/plantuml/svg/LOr1QxH038Rl-HLXhx9ywiTUYYopjfT23uLMF8zki6776PX4Ihlut-ke57e9vCdxv3MYDkeqEgX4lnoTGNbaFpon7Q8TyGxO1oyQBNkzn1W-4y8NzlIMybWUYtr__-3-vaa4_-_aCRWGiNOJ0SumG_LlxuYXcqJnq5A7zV9JA1xG2hOrB6qz2sstvlVG0BGrik2pPsNh-9lz06sJqAKJyWfW5ZrFWA-E82lpTNcDeIUH5C-A3RMMTTLhasDrFACvvVY0zsMEWiuESthMm9zmTjhaS0DJvhkN5PjPhEeCXlnr6jqF)](http://www.plantuml.com/plantuml/uml/LOr1QxH038Rl-HLXhx9ywiTUYYopjfT23uLMF8zki6776PX4Ihlut-ke57e9vCdxv3MYDkeqEgX4lnoTGNbaFpon7Q8TyGxO1oyQBNkzn1W-4y8NzlIMybWUYtr__-3-vaa4_-_aCRWGiNOJ0SumG_LlxuYXcqJnq5A7zV9JA1xG2hOrB6qz2sstvlVG0BGrik2pPsNh-9lz06sJqAKJyWfW5ZrFWA-E82lpTNcDeIUH5C-A3RMMTTLhasDrFACvvVY0zsMEWiuESthMm9zmTjhaS0DJvhkN5PjPhEeCXlnr6jqF)

See doc. here:
- https://forum.plantuml.net/14340/color-of-arrow-in-timing-diagram
- https://plantuml.com/en/timing-diagram#bb2ede32ce6accb8

Regards.
